sda-commons-server-auth-testing/src/test/java/org/sdase/commons/server/auth/testing/AuthDisabledIT.java

```diff
@@ -2,29 +2,29 @@
 
 import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
 
 import io.dropwizard.testing.ResourceHelpers;
-import io.dropwizard.testing.junit.DropwizardAppRule;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
 import javax.servlet.http.HttpServletResponse;
 import javax.ws.rs.core.Response;
-import org.junit.ClassRule;
-import org.junit.Test;
-import org.junit.rules.RuleChain;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
 import org.sdase.commons.server.auth.testing.test.AuthTestApp;
 import org.sdase.commons.server.auth.testing.test.AuthTestConfig;
 
-public class AuthDisabledIT {
+class AuthDisabledIT {
```

Comment:
I'd do it the other way round: Keep the existing test mostly like it is and create a new test for JUnit 5.